lib/dashboard-agent.js

```diff
@@ -6,39 +6,39 @@ var pusage = require("pidusage");
 var os = require("os");
 var _ = require("lodash");
 var config = require("./config");
+var socketWriterFactory = require("./socket-writer");
```

Comment:
The socket writer is not a class, just a revealing module with an exported factory function. If I start it with an uppercase letter I get the following eslint error: A function with a name starting with an uppercase letter should only be used as a constructor". If I name the variable that is returned from the require function "socketWriter", this implies that it's an instance of socket writer. Or if you want I can change it to a class.